src/measure_theory/integral/average.lean

```diff
@@ -49,14 +46,150 @@ variables {α E F : Type*} {m0 : measurable_space α}
 /-!
 ### Average value of a function w.r.t. a measure
 
-The average value of a function `f` w.r.t. a measure `μ` (notation: `⨍ x, f x ∂μ`) is defined as
-`(μ univ).to_real⁻¹ • ∫ x, f x ∂μ`, so it is equal to zero if `f` is not integrable or if `μ` is an
-infinite measure. If `μ` is a probability measure, then the average of any function is equal to its
-integral.
-
+The average value of a function `f` w.r.t. a measure `μ` (notation: `⨍ x, f x ∂μ`, `⨍⁻ x, f x ∂μ`)
```

Comment:
Can you say here how to type the `⨍` character?

Reply:
I'm not actually sure how to type it 🤔

Reply:
Ah, it doesn't seem to be in the abbreviations file in vscode-lean. Would it be possible to make a PR adding it calling it `integralbar` maybe?

Reply:
Sure I'll do that. I guess there's no point holding this PR up for it though.